.travis.yml

```diff
@@ -22,19 +22,3 @@ jobs:
       env: PURPOSE='Automation'
       script: ./scripts/ci/test_automation.sh
       python: 2.7
-    - stage: verify
-      script: ./scripts/ci/test_static.sh
-      env: PURPOSE='Static Check'
-      python: 3.6
-    - stage: verify
-      script: ./scripts/ci/test_integration.sh
-      env: PURPOSE='Integration'
-      python: 3.6
-    - stage: verify
-      script: ./scripts/ci/test_integration.sh
-      env: PURPOSE='Integration'
-      python: 2.7
-    - stage: publish
-      script: ./scripts/ci/publish.sh
-      python: 3.6
-      if: branch = master and type = push
```

Comment:
I think this is still needed. @troydai?

Reply:
@derekbekoe, I will undo this when SDK gets published
https://github.com/Azure/azure-cli-pr/issues/13